package.json

```diff
@@ -391,6 +391,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Whether to automatically send a 'hot reload' request during a Flutter debug session when saving files."
+				},
+				"dart.analyzeAngularTemplates": {
+					"type": "boolean",
+					"default": false,
+					"description": "Whether to analyze HTML files as AngularDart templates"
```

Comment:
The other preview flag is not listed in the README, so I've only added a description to the CHANGELOG